src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -157,12 +157,6 @@ public virtual async Task InvokeAsync(HttpContext context, RequestDelegate next)
                 Extensions = urlGQLRequest.Extensions ?? bodyGQLRequest?.Extensions,
                 OperationName = urlGQLRequest.OperationName ?? bodyGQLRequest?.OperationName
             };
-
-            if (string.IsNullOrWhiteSpace(gqlRequest.Query))
```

Comment:
😄 Today I came across this when trying to enable APQ on client side. I wanted to make PR, but I thought that you probably had already done this. And ta-da!

Reply:
`HandleNoQueryErrorAsync` should be eventually deleted.

Reply:
I see it in #774